MSALiOSB2C/ViewController.swift

```diff
@@ -104,7 +106,8 @@ class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate
              flow completes, or encounters an error.
              */
             
-            let parameters = MSALInteractiveTokenParameters(scopes: kScopes)
+            let webViewParameters = MSALWebviewParameters(parentViewController: self)
+            let parameters = MSALInteractiveTokenParameters(scopes: kScopes, webviewParameters: webViewParameters)
             parameters.authority = authority
             application.acquireToken(with: parameters) { (result, error) in
                 if let result = result {
```

Comment:
nit: I think guard let and do error handling first might look better, but no harm in not changing :)

Reply:
addressed